package.json

```diff
@@ -100,6 +101,7 @@
     "ember-disable-prototype-extensions": "1.1.3",
     "ember-engines": "0.8.15",
     "ember-export-application-global": "2.0.1",
+    "ember-in-viewport": "^3.10.2",
```

Comment:
This should probably be in `dependencies`, also it might be necessary to add this to the blueprint. You need to check whether the host app needs to have `ember-in-viewport` installed for it to work.

Reply:
Moved to `dependencies`. Seem to work in the host app without it installing `ember-in-viewport`.